saged/models.py

```diff
@@ -1406,3 +1506,79 @@ def fit(self, dataset: MixedDataset) -> "PseudolabelModel":
                     self.save_model(save_path)
 
         return self
+
+
+class TransferClassifier(PytorchSupervised):
+    def __init__(self,
+                 pretrained_model: PytorchImpute,
+                 optimizer_name: str,
+                 loss_name: str,
+                 model_name: str,
+                 lr: float,
+                 weight_decay: float,
+                 device: str,
+                 seed: int,
+                 epochs: int,
+                 batch_size: int,
+                 log_progress: bool,
+                 experiment_name: str = None,
+                 experiment_description: str = None,
+                 save_path: str = None,
+                 train_fraction: float = None,
+                 train_count: float = None,
+                 **kwargs,):
+        """
+        Standard model init function for a supervised model
+
+        Arguments
+        ---------
+        pretrained_model: A model trained via imputing gene expression data
+        optimizer_name: The name of the optimizer class to be used when training the model
+        loss_name: The loss function class to use
```

Comment:
I actually forgot to remove it from the parameter list. It gets provided by the calling code (the original PytorchImpute instance has to know which model it's using), but TransferClassifier itself doesn't really care since it is given a pretrained model as a parameter